src/rust/iced-x86/src/block_enc/enums.rs

```diff
@@ -7,11 +7,15 @@ use core::convert::TryFrom;
 use core::fmt;
 use core::iter::{ExactSizeIterator, FusedIterator, Iterator};
 
```

Comment:
This blank line can be removed (same thing can be said about the following files with an extra blank line)